README.md

```diff
@@ -56,8 +56,25 @@ Other features
    * Mosh leverages SSH to set up the connection and authenticate
      users. Mosh does not contain any privileged (root) code.
 
-Getting Mosh
-------------
+Getting Started
+---------------
+
+  On Ubuntu Linux
+    $ sudo apt-get install python-software-properties
+    $ sudo add-apt-repository ppa.keithw/mosh
+    $ sudo apt-get update
+    $ sudo apt-get install mosh
+
+  On fedora
+    yum install mosh
+
+  Get from git
+    $ git clone https://github.com/mobile-shell/mosh
+    $ cd mosh
+    $ ./autogen.sh
+    $ ./configure
+    $ make
+    # make install
```

Comment:
But isn't all this (and much more, in more detail) covered by the link right below to https://mosh.org/#getting ?

Reply:
To clarify, "all this" just refers to the block above, which adds instructions for Ubuntu, Fedora and building from source; while the mosh website lists these and many more. (I am not sure why you propose installing mosh from a custom PPA in Ubuntu?)